### ISSUE TITLE
manual didnt work

### ISSUE THREAD

**Issue by tofron** [mikoskinen/Blazor.Animate#17]
can you gave the code for https://animateblazorsamplessvc.azurewebsites.net/testbed ? i tried manual once="false" but i can only click once for manual to function. the other click didnt work

**Comment by atibilov** [comment]
@tofron did you find the solution?
Have a same problem with manual animation from code

**Comment by tofron** [comment]
> @tofron did you find the solution? Have a same problem with manual animation from code
nope, most of my animation now, i do it manually thru css and js.

**Comment by atibilov** [comment]
Thx
